python/lsst/obs/cfht/megacamMapper.py

```diff
@@ -191,16 +207,16 @@ def _computeCoaddExposureId(self, dataId, singleFilter):
             in which case dataId must contain filter.
         """
         tract = int(dataId['tract'])
-        if tract < 0 or tract >= 128:
-            raise RuntimeError('tract not in range [0,128)')
+        if tract < 0 or tract >= 2**MegacamMapper._nbit_tract:
+            raise RuntimeError('tract not in range [0,%d)' % (2**MegacamMapper._nbit_tract))
         patchX, patchY = list(map(int, dataId['patch'].split(',')))
         for p in (patchX, patchY):
-            if p < 0 or p >= 2**13:
-                raise RuntimeError('patch component not in range [0, 8192)')
-        id = (tract * 2**13 + patchX) * 2**13 + patchY
+            if p < 0 or p >= 2**MegacamMapper._nbit_patch:
+                raise RuntimeError('tract not in range [0,%d)' % (2**MegacamMapper._nbit_tract))
```

Comment:
I think this should be `raise RuntimeError('patch not in range [0,%d)' % (2**MegacamMapper._nbit_patch))`

Reply:
Yes good catch